### ISSUE TITLE
Running 'composer dump-autoload' causes error 

### ISSUE THREAD

**Issue by webbby** [laravel/framework#43121]
- Laravel Version: 9.2.0
- PHP Version: 8.1.1
- Database Driver & Version:

### Description:
`composer dump-autoload`  causes error 

```
In Facade.php line 178:
                                                   
Facade does not implement getFacadeAccessor method.  
                                                   

Script @php artisan package:discover --ansi handling the post-autoload-dump even
returned with error code 1
```

### Steps To Reproduce:
Create fresh install put new files into project run `composer dump-autoload`

**Comment by BrandonSurowiec** [comment]
What are these new files? It seems like you are adding your own Facade, but not declaring a binding via a 
`protected static function getFacadeAccessor() { return 'binding'; }` method.

https://laravel.com/docs/9.x/facades#main-content
![image](https://user-images.githubusercontent.com/5625680/178181942-f2c5a9c9-d22b-4efc-9e9d-28ff99e128fa.png)

**Comment by webbby** [comment]
True, some lib I didn't notice in overwritten `composer.json` did that
